tests/Task.test.ts

```diff
@@ -33,6 +33,46 @@ describe('parsing', () => {
         expect(task!.originalMarkdown).toStrictEqual(line);
     });
 
+    it('parses a task from a line (numbered)', () => {
```

Comment:
Fabulous. Thank you for writing tests.
I have been meaning to prune back some of the earlier repetitious tests that test many Task fields, instead of just the ones relevant to to the behaviour being tested.

So to not create more of that, it would really help if these tests could do the minimum that they need to do, to test the behaviour.

For example, they don't need to assert on things like due date, because they are being tested elsewhere.

Reply:
Thanks for pointing that out! I pushed a new commit with all the unnecessary checks removed.